<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminWindows</class>
 <widget class="QWidget" name="adminWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>31</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>您好，管理员！</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>370</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>101</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QPushButton" name="modPlanButton">
      <property name="text">
       <string>修改运行计划</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="delPlanButton">
      <property name="text">
       <string>删除运行计划</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="addPlanButton">
      <property name="text">
       <string>添加运行计划</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="cheUserButton">
      <property name="text">
       <string>查看用户信息</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="cheSysButton">
      <property name="text">
       <string>查看系统日志</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="updSellButton">
      <property name="text">
       <string>更新售票信息</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>551</width>
     <height>355</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page_6">
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>添加运行计划</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>270</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确认添加</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="addTable">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>391</width>
       <height>191</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>车站</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>到达时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>停车时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>里程</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>坐席1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>坐席2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>坐席3</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>车次</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="trainIdEdit">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addNewStation">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>添加车站</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>60</y>
       <width>81</width>
       <height>128</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>坐席1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="seat_1"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>坐席2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="seat_2"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>坐席3</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="seat_3"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>删除运行计划</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>车次</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>182</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>确认删除</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>70</y>
       <width>79</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>坐席1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>114</y>
       <width>79</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>坐席2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>修改运行计划</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addNewStation_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>添加车站</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="seat_4">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>176</y>
       <width>79</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="seat_5">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>132</y>
       <width>79</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>158</y>
       <width>79</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>坐席3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="seat_6">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>88</y>
       <width>79</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确认修改</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>车次</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="trainIdEdit_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="addTable_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>391</width>
       <height>191</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>车站</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>到达时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>停车时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>里程</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>坐席1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>坐席2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>坐席3</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>车次</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>日期</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>290</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>开始售票</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>290</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>停止售票</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>更新售票信息</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>车次</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>日期</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>用户信息</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>ID：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="reIdEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="usernameEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>用户日志</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>AcadEref</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>查看用户信息</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="passwordEdit"/>
     </item>
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>账号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="idEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="checkUserButton">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QTextBrowser" name="logBrowser"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>431</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>查看系统日志</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
